src/ui/toolbar/RPMIndicator.qml

```diff
@@ -0,0 +1,59 @@
+/****************************************************************************
+ *
+ * (c) 2009-2020 QGROUNDCONTROL PROJECT <http://www.qgroundcontrol.org>
+ *
+ * QGroundControl is licensed according to the terms in the file
+ * COPYING.md in the root of the source code directory.
+ *
```

Comment:
please add comment on the top describing the use of this file

Reply:
comment added #8